WordPress/gutenberg
# Link control: migrate tests to Playwright. Can be created by selecting text and using keyboard shortcuts

## What?

Part of https://github.com/WordPress/gutenberg/issues/50995
This PR migrates the test "Can be created by selecting text and using keyboard shortcuts" for the link control component to playwright

## Why?

In an effort to migrate our e2e tests to use playwright, we've decided to pick up some of the tests for this component as we work on it to improve it, instead of trying to maintain the puppeteer tests.

## How?

Following the [guidelines](https://github.com/WordPress/gutenberg/blob/971ed6c8506140421b97e38a1df77ad2666c67da/docs/contributors/code/e2e/migration.md) for e2e tests migrations

## Testing Instructions




run `npm run test:e2e:playwright -- -g "Can be created by selecting text and using keyboard shortcuts"` and the test should pass
